.circleci/config.yml

```diff
@@ -22,27 +22,21 @@ jobs:
       - checkout
 
       - save_cache:
-          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          key: v1-repo-{{ checksum "package.json" }}
           paths:
-            - ~/purposeindustries/content-editor
+            - ~/purposeindustries/intellyo-application-design-system
 
       - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
 
       - restore_cache:
           keys:
             - dependency-cache
```

Comment:
unfortunately if there's a cache with that given key, it wont update so intead of `dependency-cache` we need to use a dynamic one, can you replace the `dependency-cache` with `dependency-cache-{{ checksum "package.json" }}` everywhere?